jdaviz/core/launcher.vue

```diff
@@ -76,7 +76,7 @@
         class="mx-3"
         color="#FFFFFF"
         style="height: 180px; width: 115px"
-        @click="launch_config(config)"
+        @click="launch_config({config: config})"
```

Comment:
I have no idea how this ever worked... I'm guessing it was changed at one point but anyone testing with an editable install was not getting the latest version 🤷.  Reviewers - you may need a fresh env to force the templates to be updated in your installation.